spec/features/check.spec.ts

```diff
@@ -13,4 +13,18 @@ describe('check', () => {
       await check('not found checkbox', { timeout: 500 })
     }).rejects.toThrow()
   })
+
+  it('fails when the label points to an invalid input', async () => {
+    await check('My checkbox', { timeout: 500 })
+    await expect(async () => {
+      await check('My invalid label pointer checkbox', { timeout: 500 })
+    }).rejects.toThrow()
+  })
+
+  it('fails when the label points to an invalid input', async () => {
```

Comment:
I presume one of these is trying to check a label that doesn't exist and the other is trying to check a label that exists but doesn't point to a checkbox, but the spec example text is identical.